.github/workflows/CONTRIBUTING.md

```diff
@@ -1,10 +1,27 @@
 # Contributing to CyBear Jinni App
 
-🥳:tada: First off, thanks for taking the time to contribute! 🥳:tada:
+This documentation contains a set of guidelines to help you during the contribution process. We are happy to welcome all the contributions from anyone willing to improve/add new features to this project. Thank you for helping out, and remember, no contribution is too small.
 
-The following is a set of guidelines for contributing to CyBear Jinni App.
+# Submitting Contributions👩‍💻👨‍💻
+Below you will find the process and workflow used to review and merge your changes.
 
-These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to
-this document in a pull request.
+# Working on an Issue
+If you're interested in working on an issue, simply let us know by commenting on it. No need to wait for an assignment. We welcome your contributions!
 
-## First rule, please open pull request to dev branch and not to master
+# Fork and Pull Request
+Please fork the repository and open your pull request from the `dev` branch. This helps us maintain a clean and organized contribution process.
+
+# Link to Associated Issues
+If your pull request addresses a specific issue, make sure to include a link to that issue in the pull request description. This helps us keep track of the context and purpose of your contribution.
+# Pull Request
+- Go to your repository in browser and click on compare and pull requests. 
+- Add a title and description to your pull request that explains your contribution.
+
+- 🥳:tada: Your Pull Request has been submitted and will be reviewed by the moderators and merged.
+
+## Need more help?🤔
+
+For additional information and resources, you can refer to our [Wiki](https://github.com/CyBear-Jinni/cbj_app/wiki). It contains valuable insights and documentation that can aid you in your contributions.
```

Comment:
Let's refer here also the discord server.

You can take from the readme file the link + the logo

Reply:
Done